pym/bob/scm/url.py

```diff
@@ -203,10 +203,13 @@ def getProperties(self):
         return ret
 
     def _download(self, destination):
+        from .. import BOB_INPUT_HASH
```

Comment:
No need to lazily import it. You can move that to the import block at the top of the file.

Reply:
So this really lazy-imports? I thought this would be more about namespacing... 🤔 😁  Will do that, of course.

Reply:
When thinking about it again, it's actually quite obvious that this lazy imports, given that Python is interpreted... 🤦